dev/com.ibm.ws.logging/src/com/ibm/ws/logging/internal/impl/BaseTraceService.java

```diff
@@ -2102,4 +2177,58 @@ public void run() {
             }
         }
     }
+
+    /**
+     * FFDC task to be run/scheduled in maxFfdcAge Deletion
+     */
+    private class TimedFfdcCleanup extends TimerTask {
+        private final long maxFfdcAge;
+
+        TimedFfdcCleanup(long maxFfdcAge) {
+            this.maxFfdcAge = maxFfdcAge;
+        }
+
+        @Override
+        public void run() {
+            SimpleDateFormat sdf = new SimpleDateFormat("MM/dd/yyyy HH:mm:ss");
+            File[] postExceptionFiles = getFfdcLogs();
+            int deletedCounter = 0;
+            for (int i = 0; i < postExceptionFiles.length; i++) {
+                try {
+                    Date now = new Date();
+                    Date fileLastModifiedDate = sdf.parse(sdf.format(postExceptionFiles[i].lastModified()));
+                    long fileAge = TimeUnit.MINUTES.convert(now.getTime() - fileLastModifiedDate.getTime(),TimeUnit.MILLISECONDS);
```

Comment:
You can get current time in milliseconds from `System.currentTimeMillis()`.  Will it be more efficient to calculate the `fileAge` with `System.currentTimeMillis() - postExceptionFiles[i].lastModified()`?

Reply:
It looks like there's very little difference between them but using System.currentTimeMillis would be slightly more efficient so I'll make that change. Thanks.